### PR TITLE
add simplified chinese translation

### PR DESCRIPTION
**Pull request by Cinyan10**
#430

### CODE REVIEW

File: addons/sourcemod/translations/gokz-core.phrases.txt [hunk 1]
```diff
@@ -72,6 +73,7 @@
 	"Can't Checkpoint (Anti Checkpoint Area)"
 	{
 		"en"		"{darkred}You can't checkpoint because checkpoints are disabled in this area."
+		"en"		"{darkred}此区域禁止存点."
```
Comment (Sikarii) [inline addons/sourcemod/translations/gokz-core.phrases.txt:76]: Wrong translation key, should probably be `"chi"` instead.

File: addons/sourcemod/translations/gokz-core.phrases.txt [hunk 2]
```diff
@@ -253,7 +271,7 @@
 	}
 	"Options Menu - General"
 	{
-		"en"		"General"
+		"en"		"General"
```
Comment (Sikarii) [inline addons/sourcemod/translations/gokz-core.phrases.txt:274]: Nitpick, but remove the added whitespace.

File: addons/sourcemod/translations/gokz-core.phrases.txt [hunk 3]
```diff
@@ -185,54 +192,65 @@
...
 	}
 	"Option - Safeguard - Enable (NUB)"
 	{
 		"en"		"{grey}Safeguard for {yellow}NUB {grey}runs enabled. You can no longer issue any command that would stop your timer during a valid run."
+		"chi"		"{yellow}读点 {grey}保护模式已开启. 你不能再使用会导致计时停止的命令."
 	}
 	"Option - Safeguard - Enable (PRO)"
 	{
 		"en"		"{grey}Safeguard for {blue}PRO {grey}runs enabled. You can no longer use teleports or issue any command that would stop your timer during a valid run."
+		"en"		"{blue}裸跳 {grey}保护模式已开启. 你不能再使用传送和会导致计时停止的命令."
```
Comment (Sikarii) [inline addons/sourcemod/translations/gokz-core.phrases.txt:253]: Wrong translation key, should probably be `"chi"` instead.

File: addons/sourcemod/translations/gokz-core.phrases.txt [hunk 4]
```diff
@@ -311,21 +333,26 @@
...
 	}
 	"Timer Button Zone Type - Only End Zone"
 	{
 		"en"		"Only end zone"
+		"chi"		"仅终点自动"
 	}
 	"Timer Button Zone Type - Both Zones"
 	{
-		"en"		"Both zones"
+		"en"		"Both Zones"
```
Comment (Sikarii) [inline addons/sourcemod/translations/gokz-core.phrases.txt:345]: Revert the english translation back to the original, this PR shouldn't touch them.

**Review by Sikarii — CHANGES_REQUESTED** [review]
Thanks a lot for doing this!
I went through the changes and theres a few things that need correcting.

**Review by Sikarii — APPROVED** [review]
Looks good!

**Comment by Cinyan10** [comment]
all translations completed

**Pull request merged**